x-pack/plugins/code/server/__tests__/clone_worker.ts

```diff
@@ -238,15 +238,27 @@ describe('clone_worker_tests', () => {
       (repoServiceFactory as any) as RepositoryServiceFactory
     );
 
-    const result = await cloneWorker.executeJob({
+    const result1 = await cloneWorker.executeJob({
       payload: {
         url: 'file:///foo/bar.git',
       },
       options: {},
       timestamp: 0,
     });
 
-    assert.ok(result.repo === null);
+    assert.ok(result1.repo === null);
+    assert.ok(newInstanceSpy.notCalled);
+    assert.ok(cloneSpy.notCalled);
+
+    const result2 = await cloneWorker.executeJob({
+      payload: {
+        url: '/foo/bar.git',
```

Comment:
to address Brandon's comment on https://github.com/elastic/code/issues/1014#issuecomment-472457050